### PR TITLE
Add chihuahua chain

### PR DESCRIPTION
**Pull request by ChihuahuaChain**

### CODE REVIEW

File: cosmos/chihuahua.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        "gasPriceStep": {
+            "low": 1,
+            "average": 5,
+            "high": 10
+        }
+    }],
+    "features": [
+        "cosmwasm",
+        "ibc-transfer",
+        "ibc-go"
```
Comment (HeesungB) [inline cosmos/chihuahua.json:51]: We allow just `cosmwasm`, `secretwasm`, `eth-address-gen`, `eth-key-sign`, `axelar-evm-bridge`, `osmosis-txfees` features.

File: cosmos/chihuahua.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    ],
+    "alternativeBIP44s": [
+        {
+          "coinType": 639
+        }
+    ],
+    "walletUrl": "https://chihuahua.omniflix.co",
```
Comment (HeesungB) [inline cosmos/chihuahua.json:58]: This field is deprecated

File: cosmos/chihuahua.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    "walletUrlForStaking": "https://chihuahua.omniflix.co/stake",
+    "txExplorer": {
+        "name": "Mintscan",
+        "txUrl": "https://www.mintscan.io/chihuahua/txs/{txHash}"
+    }
```
Comment (HeesungB) [inline cosmos/chihuahua.json:63]: This field is deprecated

**Comment by HeesungB** [comment]
Hi, thank you for your contribution! We highly appreciate it.

However, there are some parts that do not comply with the requirements 
 - please remove walletURL and txExplorer fields, and ibc-transfer/ibc-go parameters in features field. 

Otherwise, everything else looks good! Thank you again, please resubmit after editing the above details.

**Comment by HeesungB** [comment]
There is one more thing to do.
Please add `chainSymbolImageUrl`

 - chainSymbolImageUrl: Image URL of the chain.
   - https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/{chain-identifier}/{file-name}.png
   - Please modify modify the chain-identifier and file-name from the link above and upload it.

**Comment by Thunnini** [comment]
@ChihuahuaChain I merged your PR. But I found that you revert the commit which has coin type 639 as alternative bips. On chihuahua chain, are there any coin types except for 118? If so, you should set them to alternative bips for users who generated the account with such coin types.

**Comment by Thunnini** [comment]
And just curious, why did you have "v1.1" appended to the chain name?

**Pull request merged**